notebooks/02_training.ipynb

```diff
@@ -0,0 +1,202 @@
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:24.860518: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:24.860813: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:24.862824: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:24.863182: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:24.863460: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:26.106820: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:26.107237: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:26.107253: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1726] Could not identify NUMA node of platform GPU id 0, defaulting to 0.  Your kernel may not have been built with NUMA support.\n",
+      "2023-08-07 18:48:26.107595: I tensorflow/compiler/xla/stream_executor/cuda/cuda_gpu_executor.cc:981] could not open file to read NUMA node: /sys/bus/pci/devices/0000:01:00.0/numa_node\n",
+      "Your kernel may have been built without NUMA support.\n",
+      "2023-08-07 18:48:26.107651: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1639] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 9330 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 4070 Ti, pci bus id: 0000:01:00.0, compute capability: 8.9\n"
+     ]
+    }
+   ],
+   "source": [
+    "NUM_VALIDATION_RECORDS = 2500\n",
+    "NUM_EPOCHS = 20\n",
+    "BATCH_SIZE = 64\n",
+    "\n",
+    "full_dataset = load_parquet_dataset(\n",
+    "    \"../data/malheur_footprints_64x64.parquet\", \n",
+    "    target_names=(\"cancov\",), \n",
+    "    shape=(64, 64)\n",
+    ")\n",
```

Comment:
Yes, that's exactly right. The `data` module was designed for my initial plan of storing columns for each band. I was updating it for the new encoded `image` column when I ran into that `tensorflow_io` bug, so that's all nonfunctional while I try and come up with a gameplan to move forward.